shared/chat/header.desktop.tsx

```diff
@@ -75,7 +75,8 @@ const Header = (p: Props) => {
       ? p.participants.filter(part => part !== p.username)
       : p.participants
 
-  if (withoutSelf && withoutSelf.length === 1 && p.desc) {
+  if (withoutSelf && withoutSelf.length === 1 && p.desc.trim()) {
+    // trim() call makes sure that string is not just whitespace
```

Comment:
nitty nit: can you put the comment above the code it refers to?

Reply:
oh yep! i had put it on the actual line but i think precommit shifted stuff around